src/components/WheelSelect/index.js

```diff
@@ -67,12 +67,13 @@ const WheelSelect = ({ options, multipleSelect }) => {
                 className="wheel-select__container"
                 key={ index }
               >
-                <div className={ elementTextClass }>{ element.display }</div>
+                <div className={ elementTextClass } data-test-id="select-display">{ element.display }</div>
 
                 { multipleSelect && (
                   <div
                     className="wheel-select__checkbox-container"
                     id={ `wheel-select__checkbox--${index}` }
+                    data-test-id="multiple-select"
```

Comment:
And this as `wheel-multiple-select`
```suggestion
                    data-test-id="wheel-multiple-select"
```